packages/elements/src/chart/elements/chart.ts

```diff
@@ -591,7 +593,7 @@ export class Chart extends BasicElement {
    * @param updateMode Additional configuration for control an animation in the update process.
    * @returns {void}
    */
-  public updateChart(updateMode: UpdateMode): void {
+  public updateChart (updateMode?: UpdateMode): void {
```

Comment:
Why add optional `?` to `updateMode` is it necessary?
Also, I don't think this syntax white space after a function name will align with the prettier config

Reply:
This change does updateChart function compatible with v4-v6. The argument should not force user to put it in. Here is a updateChart code in v6.
```
  public updateChart (config?: ChartUpdateProps): void {
```

In additional, you will see that the different the argument between v6 and v7. ChartUpdateProps and UpdateMode. I make it match with chart.js update api. [chart.js v2](https://www.chartjs.org/docs/2.9.4/developers/api.html#updateconfig) and [chart.js v4](https://www.chartjs.org/docs/latest/developers/api.html#update-mode).

Reply:
Interesting, Is this change mentioned somewhere? to make sure that we aren't missing out on any other points?

Reply:
Haven't written yet. I will update migraion grade after this merged.

Reply:
Pushed to remove unused spacebar.